### ISSUE TITLE
Error 500 when you put vue embeded v-once or v-memo directive inside v-main vuetify tag

### ISSUE THREAD

**Issue by km2442** [nuxt/nuxt#18749]
### Environment

------------------------------
- Operating System: `Windows_NT`
- Node Version:     `v16.15.0`
- Nuxt Version:     `3.1.2`
- Nitro Version:    `2.1.1`
- Package Manager:  `npm@8.5.5`
- Builder:          `vite`
- User Config:      `build`, `modules`
- Runtime Modules:  `()`
- Build Modules:    `-`
------------------------------

### Reproduction

https://stackblitz.com/edit/github-vztgqu

### Describe the bug

When you use vuetify with nuxt 3.1.2, and you put `v-once`/`v-memo` embeded vue directive inside `v-main` vuetify tag, nitro is throwing error 500 during starting app: `_cache is not defined`

### Additional context

Problem also affects production build and static generation

I've checked if it is replicable without nuxt, and no, vite + vue + vuetify is working without any problem, so **it is definitely nuxt / nitro issue**.

### Logs

```shell-script
500
_cache is not defined

at Proxy.renderFnWithContext ([path to project]\node_modules\@vue\runtime-core\dist\runtime-core.cjs.js:860:19)
at Proxy. ([path to project]\node_modules\vuetify\lib\components\VApp\VApp.mjs:47:78)
at renderComponentRoot ([path to project]\nuxt-vOnce\node_modules\@vue\runtime-core\dist\runtime-core.cjs.js:906:44)
at renderComponentSubTree ([path to project]\node_modules\@vue\server-renderer\dist\server-renderer.cjs.js:719:51)
at renderComponentVNode ([path to project]\node_modules\@vue\server-renderer\dist\server-renderer.cjs.js:644:16)
at Module.ssrRenderComponent ([path to project]\node_modules\@vue\server-renderer\dist\server-renderer.cjs.js:94:12)
at _sfc_ssrRender ([path to project]\app.js:13:31)
at renderComponentSubTree ([path to project]\node_modules\@vue\server-renderer\dist\server-renderer.cjs.js:710:17)
at renderComponentVNode ([path to project]\node_modules\@vue\server-renderer\dist\server-renderer.cjs.js:644:16)
```

**Comment by danielroe** [comment]
This seems like an upstream Vue SSR issue:

See [this playground](https://sfc.vuejs.org/#eNqlkrFugzAQhl/l5AUiEXtHhKrp3LWTF0KcQgS2ZR+0EuLde7ZJVDVDhwxY9+vuvjv/eGGv1vJ5UqxkFarRDg2qWmqA6tzPMaBw3jfWbiLKsek1zHuj21R8T5xQ18cJ0ehKJHXvIh3abkySN2gl0qxK/NqApG9dbxG8wikV9qM1DmGBs7r0Wr0Z0lppLKCDFS7OjJDRXbJQG77WaI/wQVeEw9+efEmLRHoJuXXGfhfE9oNBD+sODjXk8ezyjBbMipTiBGqmAV94vtsFxhrPbdZ7cObfYY/wR9IRnwNVIvlXs4Il48h/y6/eaPrZkSO3hJeshI0sGTkYtGQdovWlEP7Shidy9dy4T0ERd5PGflRc+XF/cubLK0dgyeJsqVe2/gAcLr+s) - Vue never initialises the `_cache` variable.

It's also not linked to Vuetify:
* [Nuxt playground without Vuetify](https://stackblitz.com/edit/github-vztgqu-ezhklc?file=app.vue,nuxt.config.ts,package.json)
* [Vue SSR playground without Vuetify](https://stackblitz.com/edit/github-7dndes?file=src%2FApp.vue,src%2Fmain.ts,package.json,vite.config.js,package-lock.json&terminal=dev)

**Comment by wiseadme** [comment]
Hi! Has this problem been fixed? Since it is still present in version 3.5.3
